src/poc/miner_onion_server_light.erl

```diff
@@ -286,14 +286,14 @@ decrypt(Type, IV, OnionCompactKey, Tag, CipherText, RSSI, SNR, Frequency, Channe
                                                         ok ->
                                                             lager:info("sending receipt with observed power: ~p with radio power ~p", [EffectiveTxPower, TxPower]),
                                                             ?MODULE:send_receipt(Data, OnionCompactKey, Type, os:system_time(nanosecond),
-                                                                                 RSSI, SNR, Frequency, Channel, DataRate, EffectiveTxPower, State);
+                                                                                 RSSI, SNR, Frequency, Channel, DR, EffectiveTxPower, State);
```

Comment:
does the spec on `send_receipt/7-8` also need to change?

Reply:
Fixed in https://github.com/helium/miner/pull/1709/commits/6248518bb89a02c095c53d51ffcc5e51d7f8ff53